edk2toollib/uefi/edk2/path_utilities.py

```diff
@@ -117,19 +119,19 @@ def GetAbsolutePathOnThisSystemFromEdk2RelativePath(self, relpath, log_errors=Tr
 
         if relpath is None:
             return None
-        relpath = relpath.replace("/", os.sep)
+        relpath = relpath.replace("/", os.sep).replace('\\', os.sep)
```

Comment:
can you add comments throughout this module (where necessary) that indicates functions will return paths using the os.sep.  

There are probably some functions that should return edk2 paths and those should be careful to always return paths with "/".